File: python/020.py
# 20) Desenvolva um programa que leia um número inteiro e mostre se ele é PAR ou
# ÍMPAR.

numero_inteiro = int(input("Digite um número: "))

if numero_inteiro % 2 == 0:
    print(f"O número {numero_inteiro} é PAR!")
else:
    print(f"O número {numero_inteiro} é ÍMPAR!")

File: javascript/020.js
/*
20) Desenvolva um programa que leia um número inteiro e mostre se ele é PAR ou
ÍMPAR.
*/

const isPar = (number) => {
  if (number % 2 === 0) return "É par";
  else return "É impar";
};

console.log(isPar(4));
console.log(isPar(6));
console.log(isPar(7));
console.log(isPar(9));
